lib/_http_outgoing.js

```diff
@@ -657,12 +657,11 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
   var output = this.output;
   var outputEncodings = this.outputEncodings;
   var outputCallbacks = this.outputCallbacks;
-  socket.cork();
+
   for (var i = 0; i < outputLength; i++) {
     ret = socket.write(output[i], outputEncodings[i],
                        outputCallbacks[i]);
   }
-  socket.uncork();
```

Comment:
You can leave the `cork()` and `uncork()` block here. Multiple calls to `cork()` and `uncork()` have no effect.